odoo/fields.py

```diff
@@ -30,15 +30,16 @@
 from .netsvc import ColoredFormatter, GREEN, RED, DEFAULT, COLOR_PATTERN
 from .tools import (
     float_repr, float_round, float_compare, float_is_zero, human_size,
-    pg_varchar, ustr, OrderedSet, pycompat, sql, SQL, date_utils, unique,
-    image_process, merge_sequences, SQL_ORDER_BY_TYPE, is_list_of, has_list_types,
+    ustr, OrderedSet, pycompat, sql, SQL, date_utils, unique,
+    image_process, merge_sequences, is_list_of,
     html_normalize, html_sanitize,
+    DEFAULT_SERVER_DATE_FORMAT as DATE_FORMAT,
+    DEFAULT_SERVER_DATETIME_FORMAT as DATETIME_FORMAT,
 )
-from .tools.misc import unquote
-from .tools import DEFAULT_SERVER_DATE_FORMAT as DATE_FORMAT
-from .tools import DEFAULT_SERVER_DATETIME_FORMAT as DATETIME_FORMAT
-from .tools.translate import html_translate, _
+from .tools.sql import pg_varchar
```

Comment:
```suggestion
from .tools.sql import pg_varchar, SQL_ORDER_BY_TYPE
```
would avoid the diff below.